lib/cli/options.js

```diff
@@ -24,6 +24,8 @@ module.exports = async function getRunOptions(appRequired, repoPathRequired) {
 
   ensurePublicConfigs(options)
 
+  ensureCompression(options)
```

Comment:
It seems like it is not necessary since it's already composed in the `enrichWithENV`, doesn't it?

Reply:
You're right, removed.